packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -101,42 +136,63 @@ export class Autocomplete extends React.PureComponent {
       loading,
       children,
       className,
+      noClearInput,
       ...autocompleteProps
     } = this.props;
 
+    // Custom container returns a plain div, without the ARIA markup
+    // required for a WAI-ARIA 1.1 combobox. See the comments at the
+    // top of the component file for an explanation of this decision.
+    const ComponentDiv = ({ innerRef, ...rest }) => (
+      <div ref={innerRef} {...rest} />
+    );
+
     const rootClassName = classNames(
       'ds-u-clearfix',
       'ds-c-autocomplete',
       className
     );
 
     return (
-      <Downshift
-        render={({
+      <Downshift {...autocompleteProps}>
+        {({
           clearSelection,
           getInputProps,
           getItemProps,
+          getRootProps,
           highlightedIndex,
           inputValue,
           isOpen
         }) => (
-          <div className={rootClassName}>
+          <ComponentDiv
```

Comment:
I don't entirely understand what's going on here, but would it be possible to put these props directly on an inline `<div ...` declaration rather than creating the separate `ComponentDiv` function above? If it's a matter of pulling out the `innerRef` property, we could change this render function a bit to do something like
```js
(stuff) => {
  const rootProps = getRootProps({ ...blahblahblah });
  return (
    <div {...rootProps} ref={rootProps.innerRef}>
    </div>
  );
}
```
If what `refKey: 'innerRef'` is doing is telling Downshift what to call the property, could we say `refKey: 'ref'` and not have to do anything special to apply it to the `ref` prop?

Reply:
Downshift looks for a `<div>` as the container element, and applies a lot of ARIA attributes to make the finished component fall in-line with the ARIA 1.1 specification for comboboxes. I discovered in screen reader testing this is unpredictable and introduces a number of challenges for usability. 

I opted to use the previous pattern ( https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.0pattern/combobox-autocomplete-list.html ) and needed a way to suppress those ARIA attributes. Using `getRootProps` looked like the best way, and I modeled the `<MyDiv />` "component" after prior art on the Downshift issues page. ( https://github.com/downshift-js/downshift/issues/235)